File: JavaZero/app/src/main/java/com/yagofellipe/javazero/MainActivity.java
package com.yagofellipe.javazero;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.yagofellipe.javazero.classes.ContaBancaria;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ContaBancaria conta = new ContaBancaria();
    }
}

File: app/src/main/java/com/yagofellipe/imc/MainActivity.kt
package com.yagofellipe.imc

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {

    private lateinit var btnCalcular: Button
    private lateinit var editPeso: EditText
    private lateinit var editAltura: EditText
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnCalcular = findViewById(R.id.btn_calcular)
        editPeso = findViewById(R.id.edit_peso)
        editAltura = findViewById(R.id.edit_altura)

        btnCalcular.setOnClickListener {

            val intent = Intent(this, ResultadoActivity::class.java)

            val peso = editPeso.text.toString()
            val altura = editAltura.text.toString()

            if (peso.isNotEmpty() && altura.isNotEmpty()){
                intent.putExtra("peso",peso.toDouble())
                intent.putExtra("altura",altura.toDouble())
            }

            startActivity(intent)
        }
    }
}